iroh-base/src/ticket/node.rs

```diff
@@ -10,9 +10,14 @@ use crate::{
     ticket::{self, Ticket},
 };
 
-/// A token containing everything to get a file from the provider.
+/// A token containing information for dialing a node.
+///
+/// Contains
+/// - The [`NodeId`] of the node to connect to (a 32-byte ed25519 public key)
```

Comment:
I love adding docs, though this is a list with one item.  Did you mean to add some more things?